components/faucet/FormComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { Input, InputGroup, Spinner, chakra, useToast } from "@chakra-ui/react";
+import axios from "axios";
+import { FC, useState } from "react";
+import { Button } from "tw-components";
+
+const FormComponent: FC = () => {
```

Comment:
```suggestion
export const FormComponent: React.FC = () => {
```
same as the other component